docs/manual/guides/deployer.en.md

```diff
@@ -0,0 +1,278 @@
+---
+title: "Deploy with Deployer"
+description: "Learn how to deploy your project with Deployer."
+aliases:
+    - /en/guides/deployer/
+weight: 80
+tags: 
+    - "Deployer"
+    - "Deployment"
+---
+
+{{% notice note %}}
+The Deployer recipe is part of Deployer 7 and is intended to work for Contao 4.13 and higher.
+
+## Install Deployer
+
+If not done yet, install Deployer as described here: [https://deployer.org/docs/][1]
+
+Verify that you are running Deployer in the minimum version _7.0.0-rc.5_ by running `dep --version`.
+
+Once done, you can create a `deploy.php` file in your project:
+
+```bash
+touch deploy.php
+```
```

Comment:
Alternatively you can also execute

```
composer require deployer/deployer:^7.0.0-rc.5 --dev
```

and then use `vendor/bin/dep`. This is what I prefer, so that you don't having to worry about installing global dependencies for specific projects (same as with CS fixer or phpunit).

Reply:
I think this is already covered by the Deployer docs.

Reply:
Hm, https://deployer.org/docs/7.x/getting-started does not mention it.

Reply:
Changed the link to https://deployer.org/docs/7.x/installation in 876954a14d3323e62ec31e282d460cf8a5df152a. This page mentions that the local dist installation method is the preferred one.

Reply:
We just might to review the usages of `dep` and use `[vendor/bin/]dep`.

Reply:
d169f1b6b0f33803bf949fe60ab6eed31e973685.